netkan/netkan/scheduler.py

```diff
@@ -25,29 +27,24 @@ def netkans(self):
         # implementing like for like initially.
         return (Netkan(f) for f in self.path.glob('*.netkan'))
 
-    def sqs_batch_entries(self, batch_size=10):
-        batch = []
-
-        for netkan in self.netkans():
-            if self.force_all or not netkan.hook_only():
-                batch.append(netkan.sqs_message())
-                if len(batch) == batch_size:
-                    yield(batch)
-                    batch = []
-        if len(batch):
-            yield(batch)
-
     def sqs_batch_attrs(self, batch):
         return {
             'QueueUrl': self.queue_url,
             'Entries': batch
         }
 
+    def _in_group(self, netkan):
+        if netkan.hook_only():
+            return self.webhooks_group
+        else:
+            return self.nonhooks_group
+
```

Comment:
I'm not overly sure about this logic, but we can add some tests for this and re-factor later if need be. Much better than what was springing to mind when I last had a look and this does the job. :+1: